README.md

```diff
@@ -1,25 +1,33 @@
 # Package Storage
-This is a storage repository for the packages served through the [package registry](https://github.com/elastic/package-registry) service.
+This is a storage repository for the packages served through the [package registry](https://github.com/elastic/package-registry) service.  See the package-registry repo for basic registry api usage and examples.
 
-It contains 3 branches with the packages for the different environments:
+Package storage repo contains 3 branches with the packages for the different environments:
 
 * snapshot
 * staging
 * production
 
 Here's how these branches relate to repositories and other aspects of packages.
 
-|                       | Snapshot                    | Staging                    | Production                |
-|-------------------    |-------------------------    |------------------------    |-----------------------    |
-| URL                   | epr-snapshot.elastic.co     | epr-staging.elastic.co     | epr.elastic.co            |
-| Add package           | Commit                      | Commit                     | PR                        |
-| Version overwrite     | yes                         | if needed                  | no                        |
-| Stack Version         | *-SNAPSHOT                  | *-SNAPSHOT                 | Released (candidates)     |
-| Registry Version      | fixed version                | Stable release             | Stable release            |
-| Branch                | snapshot                    | staging                    | production                |
-| Packages              | snapshot+staging+prod       | staging+production         | production                |
-| Release               | Manual                      | Manual                  | Manual                 |
-| Docker image          | snapshot                    | staging                    | production                |
+|                         | Snapshot                    | Staging                    | Production                |
+|-------------------      |-------------------------    |------------------------    |-----------------------    |
+| URL                     |  [epr-snapshot.elastic.co](https://epr-snapshot.elastic.co/search) | [epr-staging.elastic.co](https://epr-staging.elastic.co/search) | [epr.elastic.co](https://epr.elastic.co/search) |
+| How to Add package      | Commit to elastic/integrations* | PR with elastic-package tool | PR with elastic-package tool |
```

Comment:
```suggestion
| How to add a package      | Commit to elastic/integrations* | [`elastic-package promote`](https://github.com/elastic/elastic-package#elastic-package-promote) | [`elastic-package promote`](https://github.com/elastic/elastic-package#elastic-package-promote) |
```

Reply:
I would keep commit here. As `elastic/integrations` is just one place where packages can come from.

Reply:
@ruflin in that case, should it be `commit` for `snapshot` but `PR` for `staging` and `production`? I don't think we want folks making direct commits to `staging`, right?

Reply:
Good point. I would go further and say we it always PR. It is more a question if PR's are auto merged or not. Today also to snapshot it is a PR.

Reply:
so... what do we want here?  feel free to make a formal suggestion.  I accounted for 'integrations' not being the only place commits come from with the asterisk note.  It will be the most common, won't it?  Endpoint being the only current exception, right?  I'd like to keep the doc as helpful and detailed to the 99% of intended audience.  

let me know if you want my edit, or Shuanak's or something new.

Reply:
To not block this PR, just take the suggestion from @ycombinator and we will do a follow up PR / discussion.